clients/hadoopfs/src/test/java/io/lakefs/BulkDeleterTest.java

```diff
@@ -0,0 +1,101 @@
+package io.lakefs;
+
+import io.lakefs.clients.api.*;
+import io.lakefs.clients.api.model.*;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Test;
+import org.hamcrest.CustomTypeSafeMatcher;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.io.IOException;
+
+public class BulkDeleterTest {
+    protected ExecutorService executorService = Executors.newSingleThreadExecutor();
+
+    @After
+    public void shutdownExecutor() {
+        executorService.shutdown();
+    }
+
+    @Test
+    public void nothing() throws IOException {
+        BulkDeleter deleter = new BulkDeleter(executorService, new BulkDeleter.Callback() {
+                public ObjectErrorList apply(String repository, String branch, PathList pathList) throws ApiException {
+                    throw new ApiException("failed for testing");
+                }
+            },
+            "repo", "branch", 50);
```

Comment:
```suggestion
         BulkDeleter deleter = new BulkDeleter(executorService, (repo, branch, pathList) -> {
            throw new ApiException("failed for testing");
        }, "repo", "branch", 50);
```
Any reason for us not to support Lambdas at this point?